pkg/client/clientcmd/loader.go

```diff
@@ -57,14 +63,29 @@ type ClientConfigLoadingRules struct {
 // NewDefaultClientConfigLoadingRules returns a ClientConfigLoadingRules object with default fields filled in.  You are not required to
 // use this constructor
 func NewDefaultClientConfigLoadingRules() *ClientConfigLoadingRules {
+	chain := []string{}
+	migrationRules := map[string]string{}
+
+	envVarFiles := os.Getenv(RecommendedConfigPathEnvVar)
+	if len(envVarFiles) != 0 {
+		chain = append(chain, strings.Split(envVarFiles, ",")...)
```

Comment:
Depending on the OS.  I believe path/filepath has a helper though

http://golang.org/pkg/path/filepath/#Split

Cross OS safe

> On Apr 15, 2015, at 9:48 PM, Brian Grant <notifications@github.com> wrote:
> 
> In pkg/client/clientcmd/loader.go:
> 
> > @@ -57,14 +63,29 @@ type ClientConfigLoadingRules struct {
> >  // NewDefaultClientConfigLoadingRules returns a ClientConfigLoadingRules object with default fields filled in.  You are not required to
> >  // use this constructor
> >  func NewDefaultClientConfigLoadingRules() *ClientConfigLoadingRules {
> > +	chain := []string{}
> > +	migrationRules := map[string]string{}
> > +
> > +	envVarFiles := os.Getenv(RecommendedConfigPathEnvVar)
> > +	if len(envVarFiles) != 0 {
> > +		chain = append(chain, strings.Split(envVarFiles, ",")...)
> Colon would be similar to PATH.
> 
> —
> Reply to this email directly or view it on GitHub.
>